frontend/app/components/comment-form/comment-form.tsx

```diff
@@ -15,6 +15,7 @@ import { SubscribeByRSS } from './__subscribe-by-rss';
 
 import MarkdownToolbar from './markdown-toolbar';
 import TextareaAutosize from './textarea-autosize';
+import { TextExpander } from './text-expander';
```

Comment:
Maybe we can make a chunk and load it only when emoji is enabled?

Reply:
expander component about 3k (https://bundlephobia.com/result?p=@github/text-expander-element@1.0.1). node-emojis will load only on first search. Of course it can be improved. But in current state will be better to move comment form to separate chunk. But it out of the scope this pull request.

Reply:
Yes of course.